spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/secretmanager/SecretManagerPropertySource.java

```diff
@@ -48,12 +47,14 @@ public SecretManagerPropertySource(
 			String propertySourceName,
 			SecretManagerServiceClient client,
 			GcpProjectIdProvider projectIdProvider,
-			String secretsPrefix) {
+			String secretsPrefix,
+			String version,
```

Comment:
This change to the constructor would be breaking. Can we instead add an additional constructor with extra version params?